android/src/main/java/com/rcttabview/RCTTabViewViewManager.kt

```diff
@@ -66,6 +68,13 @@ class RCTTabViewViewManager :
   fun setIcons(view: ReactBottomNavigationView, icons: ReadableArray?) {
     view.setIcons(icons)
   }
+  @ReactProp(name = "rippleColor")
+  fun setRippleColor(view: ReactBottomNavigationView, rippleColor: Int?){
+    if(rippleColor!=null) {
+      val color = ColorStateList.valueOf(rippleColor)
+      view.setRippleColor(color)
+    }
+  }
```

Comment:
Something is off with formating here. Can you reformat the code with Android Studio?